packages/eslint-plugin/src/rules/my-first-rule.ts

```diff
@@ -0,0 +1,23 @@
+import type { Rule } from "eslint";
```

Comment:
Do we still need this my first rule?

Reply:
I've kept it for reference/template but we can remove it. It's not enabled anyways.